src/app/bisoncoin/src/App.vue

```diff
@@ -1,22 +1,24 @@
 <template>
   <div id="app">
-    <LoginPage msg="Welcome to BisonCoin!" />
+    <router-view />
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
 import LoginPage from "./pages/LoginPage.vue";
+import HomePage from "./pages/HomePage.vue";
 
 @Component({
   components: {
     LoginPage,
+    HomePage,
   },
 })
 export default class App extends Vue {}
 </script>
 
-<style>
+<style lang="scss">
 #app {
```

Comment:
Do we use any css for App.vue? I think this might have been put here when the project was created. If it is not used we can probably remove all the css lines we don't need!

Reply:
Good point, fixed!